sdk/test/common/BUILD

```diff
@@ -93,3 +93,14 @@ cc_test(
         "@com_google_googletest//:gtest_main",
     ],
 )
+
+cc_test(
+    name = "attribute_utils_test",
+    srcs = [
+        "attribute_utils_test.cc",
+    ],
+    deps = [
+        "//sdk/src/trace",
```

Comment:
This likely only need to depend on `//sdk/src/common`, I don't see anything trace specific in the test.

Reply:
Yup good catch, fixed it to just use ` //sdk:headers` which follows what is done for `atomic_unique_ptr_test`